### DISCUSSION TITLE
Octocat isn't in the middle.

### DISCUSSION THREAD

**Discussion by dongho-jung** [community/community#38161] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/19762154/199701527-4af7dfa4-f24c-40b5-ba9e-77157955fc67.png)

![image](https://user-images.githubusercontent.com/19762154/199701569-eb75871b-7870-4329-833e-3fc205cefcd1.png)

![image](https://user-images.githubusercontent.com/19762154/199701892-ff13cfcd-f5c1-4947-a04e-11d52180d411.png)

maybe it should use also flex-items-center not only flex-justify-center I guess..

**Comment by senith-sulara** [comment]
You can use 
<img src="" alt="" class="center">
or in css file 

.center {
  display: block;
  margin-left: auto;
  margin-right: auto;
  width: 50%;
}

or try to put image in center tag 

 <center>
        <div style=""  >
            <img src="" alt="" />
        </div>
 </center>
